src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Operations/Reindex/ReindexJobTaskTests.cs

```diff
@@ -214,6 +214,7 @@ public async Task GivenRunningJob_WhenExecuted_ThenQueuedQueryCompleted()
 
             Assert.Equal(OperationStatus.Completed, job.Status);
             Assert.Equal(10, job.Count);
+            Assert.Equal(4, job.Progress);
```

Comment:
>Assert.Equal(4, job.Progress); [](start = 12, length = 30)

Just to note: `PercentComplete` here returns 40 (it's defined as Progress/Count *100);
For person new to project it seems a bit odd to have operation status = Completed, but `PercentComplete` is not 100.

Reply:
It is true, it is a result of the faked data for the test, where fakes are not entirely consistent.